install_config/configuring_sdn.adoc

```diff
@@ -183,6 +192,27 @@ networkConfig:
 *redhat/openshift-ovs-networkpolicy* for the *ovs-networkpolicy* plug-in
 ====
 
+[[growing-the-service-network]]
+== Growing the Service Network
```

Comment:
Would s/growing/expanding make more sense? I feel like "growing" isn't the best word because it's not growing itself, but you're giving it the capability to expand?